source/video_core/src/video_core/renderer.hpp

```diff
@@ -64,7 +64,7 @@ class Renderer {
      */
     virtual bool BlitImage(Context&, uint32_t /* TODO: PAddr */ input_addr, uint32_t input_width, uint32_t input_height, uint32_t input_stride,
                            uint32_t input_format, uint32_t output_addr, uint32_t output_width, uint32_t output_height,
-                           uint32_t output_stride, uint32_t output_format) = 0;
+                           uint32_t output_stride, uint32_t output_format, uint32_t flip_y) = 0;
```

Comment:
There's only two possible values for this, so a boolean should be used:
```suggestion
                           uint32_t output_stride, uint32_t output_format, bool flip_y) = 0;
```